test/trackerBlocking.js

```diff
@@ -143,12 +177,14 @@ describe('generateTrackerBlockingRuleset', () => {
 
     it('should notice missing isRegexSupported argument', async () => {
         await assert.rejects(() =>
+            // @ts-expect-error
```

Comment:
Can you explain why you're expecting this error...
```suggestion
            // @ts-expect-error - something something descriptive
```

Reply:
Good idea, Done.